LibreNMS/OS/Fabos.php

```diff
@@ -118,4 +124,43 @@ protected function fcSwitchModelToName($model)
 
         return $models[$model] ?? 'Unknown Brocade FC Switch';
     }
+
+    public function discoverTransceivers(): Collection
+    {
+        //Only used the 2 below lines to allow "collect_snmp_data" to do the collection
+        //for some reason, it does not collect the SnmpQuery
+        //$unused = snmp_walk($this->getDeviceArray(), 'swConnUnitPortTable', '-OQXUt', 'FA-EXT-MIB');
```

Comment:
Question: For some reason, the SNMP data was not collected in the SnmpQuery below. Which means the transceivers module did not appear in tests. I had to us the deprecated functions snmp_walk to allow the collect_snmp_data to "see" those and collect them. Now tests are running correctly. 
Any clue why ?